cyral/resource_cyral_repository_local_account_test.go

```diff
@@ -418,6 +418,7 @@ func TestAccRepositoryLocalAccountResource_AwsIAM(t *testing.T) {
 				Config: testUpdatedConfig,
 				Check:  testUpdatedCheck,
 			},
+			// TODO: add import test -aholmquist 2022-08-01
```

Comment:
I think we can test the import for those resources that have a complex ID by using the `ImportStateId`. I'm ok to do this in a separate PR later, as this only affects the automated tests and we already tested the resources manually.